app/models/logstash_queries/elastic_link_popularity_query.rb

```diff
@@ -15,6 +15,7 @@ def body
           json.filter do
             json.bool do
               json.must do
+                json.child! { json.term { json.type 'click' } }
```

Comment:
This adds the `"type": "click"` filter to the ES query itself, which is necessary now that we cannot pass the type as an argument to the ES client.